script/update-cruby

```diff
@@ -24,7 +26,17 @@ else
   exit 1
 fi
 
+openssl_url="https://www.openssl.org/source/openssl-${openssl_version}.tar.gz"
+if command -v sha256sum >/dev/null; then
+  openssl_sha256=$(sha256sum "$release_directory/$openssl_basename" | cut -d ' ' -f 1)
```

Comment:
I think you need to download the openssl archive before this line, otherwise it won't be there

Reply:
Ah it's done in the workflow, I see. That doesn't seem practical if this script is ever run manually though.